subset/security/brute_test/src/main/java/SetupTest.java

```diff
@@ -0,0 +1,60 @@
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SetupTest {
+
+  String host;
+  int connectionPort;
+  String macAddress;
+  SshSocket sshSocket = null;
+  Map<String, String> macDevices = new HashMap<String, String>();
+  boolean debug = false;
+  static final int minimumMACAddressLength = 5;
+
+  public void readLocalFile() {
+    try {
+      InputStream inputStream = this.getClass().getResourceAsStream("/macList.txt");
+      StringBuilder resultStringBuilder = new StringBuilder();
+      BufferedReader br = new BufferedReader(new InputStreamReader(inputStream));
+      String line;
+      while ((line = br.readLine()) != null) {
+        resultStringBuilder.append(line).append("\n");
+        String macAddress;
+        String manufacturer;
+        if (line.length() > minimumMACAddressLength) {
+          macAddress = line.substring(0, 6);
```

Comment:
It'd be great if 0/6/7 were named constants (not a big thing)

Reply:
should be constants trevor will go nuts for this